src/style.css

```diff
@@ -9,6 +9,17 @@
   src: url(fonts/Avenir-Heavy.ttf);
 }
 
+* {
+  box-shadow: 1px 1px 8px rgba(61, 78, 97, 0.1);
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+  box-sizing: border-box;
```

Comment:
this https://github.com/mate-academy/layout_search-bar-airbnb/pull/2977#discussion_r1120376319 was only about `box-sizing`, i.e now you should move `box-shadow` and `font-weight` back to  `.searchInput` (keep only box-sizing in `*`)

P.S. and probably move `font-family: Avenir, sans-serif;` to `.searchInput` as well if with `font-family: inherit` it won't work as expected

Reply:
now it's right?